samples/ios-unified/PushSettingsViewController.cs

```diff
@@ -20,7 +20,7 @@ public override void ViewDidLoad()
 
             // Initialize switches
             this.pushEnabledSwitch.On = UAirship.Push().UserPushNotificationsEnabled;
-            locationEnabledSwitch.On = UAirship.Location().LocationUpdatesEnabled;
+            locationEnabledSwitch.On = UAirship.Shared().LocationProviderDelegate.GetLocationUpdatesEnabled();
```

Comment:
Did you get this working?

Reply:
Nah. It won't function until I add the LocationKit as a new package. That's my next PR.